src/components/publiq-ui/Button.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types';
+import { Button as BootstrapButton } from 'react-bootstrap';
+
+const ButtonVariants = {
+  PRIMARY: 'primary',
+  SECONDARY: 'outline-secondary',
+  SUCCESS: 'success',
+  DANGER: 'danger',
+};
+
+const Button = ({ variant, disabled, loading, children }) => {
+  console.log(loading);
```

Comment:
I think a debug statement was forgotten here. Maybe something that could be spotted by a linter?

Reply:
I left it here because ESLint was complaining, these components still need some internal work